airflow/www/templates/airflow/dag_details.html

```diff
@@ -29,7 +29,7 @@ <h2>{{ title }}</h2>
       <a
           class="btn"
           style="border: none; background-color:{{ State.color(state)}}; color: {{ State.color_fg(state) }};"
-          href="/taskinstance/list/?_flt_3_dag_id={{ dag.dag_id }}&_flt_3_state={{ state }}">
+          href="{{ url_for('TaskInstanceModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}&_flt_3_state={{ state }}">
```

Comment:
IIRC `url_for` accepts arbitrary `**kwargs` to build a query string, so we should be able to write `{{ url_for('TaskInstanceModelView.list', _flt_3_dag_id=dag.dag_id, _flt_3_state=state) }}` instead of building the url in the template.

Reply:
I apply your suggestion.  I will introduce similar changes in other places in the project as seperate PR. I add your suggestion to the list of my tasks.

Reply:
I would like to thank you for taking your time to review this change. Your suggestions are very important and valuable to me.